tests/milvus_memory_test.py

```
@@ -68,5 +68,5 @@ def test_get_stats(self) -> None:
             stats = self.memory.get_stats()
             self.assertEqual(15, len(stats))
 
-except:
+except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)